lib/surrealist/string_utils.rb

```diff
@@ -19,11 +19,12 @@ class << self
       #
       # @return [String] new underscored string.
       def underscore(string)
-        string.gsub(NAMESPACES_SEPARATOR, UNDERSCORE)
-          .gsub(DASH_REGEXP1, UNDERSCORE_SUBSTITUTE)
-          .gsub(DASH_REGEXP2, UNDERSCORE_SUBSTITUTE)
-          .tr(DASH, UNDERSCORE)
-          .downcase
+        new = string.gsub(NAMESPACES_SEPARATOR, UNDERSCORE)
```

Comment:
can we name this local variable `dup` instead of `new`?

Reply:
@nesaulov done!